src/session/session_dhandle.c

```diff
@@ -733,7 +733,7 @@ __session_get_dhandle(WT_SESSION_IMPL *session, const char *uri, const char *che
     }
 
     /* Sweep the handle list to remove any dead handles. */
-    __session_dhandle_sweep(session);
+    __wt_session_dhandle_sweep(session);
```

Comment:
Not related to this change but it is the end of the change: I think the test you wrote should also be added to this.